static/src/javascripts-legacy/projects/common/modules/gallery/lightbox.js

```diff
@@ -1,538 +0,0 @@
-                    this.translateContent(this.$slides.length, 0, 0);
-                },
-                'close': function () { this.state = 'closed'; }
-            }
-        }
-    };
-
-    GalleryLightbox.prototype.show = function () {
-        var $body = bonzo(document.body);
-        this.bodyScrollPosition = $body.scrollTop();
-        $body.addClass('has-overlay');
-        this.$lightboxEl.addClass('gallery-lightbox--open');
-        bean.off(document.body, 'keydown', this.handleKeyEvents); // prevent double binding
-        bean.on(document.body, 'keydown', this.handleKeyEvents);
-    };
-
-    GalleryLightbox.prototype.close = function () {
-        if (url.hasHistorySupport) {
-            url.back();
-        } else {
-            this.trigger('close');
-        }
-        this.trigger('close');
-    };
-
-    GalleryLightbox.prototype.hide = function () {
-        // remove has-overlay first to show body behind lightbox then scroll and
-        // close the lightbox at the same time. this way we get no scroll flicker
-        var $body = bonzo(document.body);
-        $body.removeClass('has-overlay');
-        bean.off(document.body, 'keydown', this.handleKeyEvents);
-        window.setTimeout(function () {
-            if (this.bodyScrollPosition) {
-                $body.scrollTop(this.bodyScrollPosition);
-            }
-            this.$lightboxEl.removeClass('gallery-lightbox--open');
-            mediator.emit('ui:images:vh');
-        }.bind(this), 1);
-    };
-
-    GalleryLightbox.prototype.pulseButton = function (button) {
-        var $btn = bonzo(button);
-        $btn.addClass('gallery-lightbox__button-pulse');
-        window.setTimeout(function () { $btn.removeClass('gallery-lightbox__button-pulse'); }, 75);
-    };
-
-    GalleryLightbox.prototype.handleKeyEvents = function (e) {
-        if (e.keyCode === 37) { // left
-            this.trigger('prev');
-        } else if (e.keyCode === 39) { // right
-            this.trigger('next');
-        } else if (e.keyCode === 38) { // up
-            this.trigger('show-info');
-        } else if (e.keyCode === 40) { // down
-            this.trigger('hide-info');
-        } else if (e.keyCode === 27) { // esc
-            this.close();
-        } else if (e.keyCode === 73) { // 'i'
-            this.trigger('toggle-info');
-        }
-    };
-
-    GalleryLightbox.prototype.endslate = new Component.Component();
-
-    GalleryLightbox.prototype.loadEndslate = function () {
-        if (!this.endslate.rendered) {
-            this.endslateEl = bonzo.create(endslateTpl);
-            this.$contentEl.append(this.endslateEl);
-
-            this.endslate.componentClass = 'gallery-lightbox__endslate';
-            this.endslate.endpoint = '/gallery/most-viewed.json';
-            this.endslate.prerender = function () {
-                bonzo(this.elem).addClass(this.componentClass);
-            };
```

Comment:
Is the `prerender` functionality longer needed?

Reply:
Yes! Not sure how I lost it. I brought it back though :)